sdk/src/main/java/com/hedera/hashgraph/sdk/NetworkName.java

```diff
@@ -0,0 +1,39 @@
+package com.hedera.hashgraph.sdk;
+
+public enum NetworkName {
+    MAINNET(0),
+    TESTNET(1),
+    PREVIEWNET(2);
```

Comment:
Since users do not have access to the underlying data of each enumeration variant I'm going to just go with just an integer. I think we'll be able to update to be a byte array later without an issue.